app/views/home/index.html.erb

```diff
@@ -33,12 +34,11 @@
       </div>
     </div>
   </div>
-  <a id="services" class="anchor"></a>
   <div class="row works-container">
```

Comment:
Why is this called works-container but the anchor is called work?

Reply:
To match the navbar button